i18n-config.js

```diff
@@ -4,32 +4,38 @@ module.exports = {
         is_default: true,
         path: 'en',
         display_name: 'English',
-        short_name: 'ENG'
+        short_name: 'ENG',
+        affiliate_lang: 0,
```

Comment:
affiliate_lang?

Reply:
affiliate pages have different way of showing language, it uses numeric value, i tried to map it here